packages/feedback-spa/src/app/apollo.config.ts

```diff
@@ -23,77 +23,9 @@ export class GraphQLModule {
   constructor(apollo: Apollo, httpLink: HttpLink) {
 
```

Comment:
If you are getting __typename in the api response you need to use the hlper method called omitTypename